test/gui/tst_syncing/test.feature

```diff
@@ -238,9 +234,9 @@ Feature: Syncing files
 
 
     Scenario: Verify pre existing folders in local (Desktop client) are copied over to the server
-        Given the user has created a folder "Folder1"
-        And the user has created a folder "Folder1/subFolder1"
-        And the user has created a folder "Folder1/subFolder1/subFolder2"
+        Given user "Alice" has created a folder "Folder1" inside the sync folder
+        And user "Alice" has created a folder "Folder1/subFolder1" inside the sync folder
+        And user "Alice" has created a folder "Folder1/subFolder1/subFolder2" inside the sync folder
```

Comment:
Without adding a user to the client, the user sync path will be unknown.
So while creating files/folders before starting the client, the user should be mentioned so that the files/folders are created in the correct folder.